package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@observablehq/framework",
   "license": "ISC",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Please revert this change. We’ll bump the version when we prepare a release. Thank you.